### ISSUE TITLE
Menu items wrongly displayed

### ISSUE THREAD

**Issue by hectorperez** [cotes2020/jekyll-theme-chirpy#1268]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Generated from `chirpy-starter`

### Describe the bug

In the menu bar of https://chirpy.cotes.page, and in my local installation, the menu items are not displayed properly. 

For example, have a look at "Archive" and "About" in the menu at the left of the following screenshot:
![chirpy](https://github.com/cotes2020/jekyll-theme-chirpy/assets/9378/243012e5-71d3-488e-bcf7-f34337690657)

When you move the mouse over the text, it gets fixed. But when you leave, some get broken again.

### Steps To Reproduce

1. Go to https://chirpy.cotes.page
2. Look at "Archive" and "About" in the left menu bar

### Expected Behavior

The text should be clear to read.

### Environment

On https://chirpy.cotes.page you can reproduce the error.

In local too:
- Ruby: 3.2.2
- Jekyll: 4.3.2
- Chirpy: jekyll-theme-chirpy (6.2.2)

### Anything else?

Thanks!

**Comment by hectorperez** [comment]
The problem seems to be my version of Chrome. In Firefox, and other Chrome browsers, it works.

**Comment by hectorperez** [comment]
I'm reopening this as it's still an issue.

Chrome version 117.0.5938.92
Ubuntu 23.04

I disabled the extensions and tried in incognito mode.
Someone else reported the same problem too. Although it's ok for others.

Removing the body font-family "Source Sans Pro","Microsoft Yahei",sans-serif;` looks right.
It could be an issue with Ubuntu default fonts — which I haven't changed.

**Comment by cotes2020** [comment]
You might consider raising an issue with Chrome.

**Comment by hectorperez** [comment]
Thanks @cotes2020.

Meanwhile, in order to replace the font family in my blog, how could I redefine `$font-family-base` from chirpy's `_sass/addon/variables.scss` in my blog? Thanks!

**Comment by hectorperez** [comment]
@cotes2020 I tried that font family in a simple html file [1] and it's displayed properly in Chrome. Yet, my Chirpy site and https://chirpy.cotes.page don't.

By the way, even if it were a problem of Chrome, it would still be an issue for everyone with old Chrome versions.
So it would be great to have a way to redefine `$font-family-base` in my project.

I tried to redefine `variables.scss` but it didn't work.
Should it be placed in the directory `sass`, `_sass` or `sass/addons`?
What do I need to add to `_config.yml` under `sass:`? 

Thanks!

[1] 
```html
<!DOCTYPE html>
<html>
  <body>
    <p>test</p>
    <p style='font-family: "Source Sans Pro","Microsoft Yahei",sans-serif;'>test</p>
    <p style='font-family: "Arial, sans-serif;'>test</p>
  </body>
</html>
```
